miner/algo_common.go

```diff
@@ -240,15 +247,23 @@ func (envDiff *environmentDiff) commitTx(tx *types.Transaction, chData chainData
 
 // Commit Bundle to env diff
 func (envDiff *environmentDiff) commitBundle(bundle *types.SimulatedBundle, chData chainData, interrupt *int32, algoConf algorithmConfig) error {
-	coinbase := envDiff.baseEnvironment.coinbase
-	tmpEnvDiff := envDiff.copy()
+	var (
+		coinbase   = envDiff.baseEnvironment.coinbase
+		tmpEnvDiff = envDiff.copy()
+
+		coinbaseBalanceBefore = tmpEnvDiff.state.GetBalance(coinbase)
 
-	coinbaseBalanceBefore := tmpEnvDiff.state.GetBalance(coinbase)
+		profitBefore = new(big.Int).Set(tmpEnvDiff.newProfit)
 
-	profitBefore := new(big.Int).Set(tmpEnvDiff.newProfit)
-	var gasUsed uint64
+		gasUsed uint64
+	)
 
 	for _, tx := range bundle.OriginalBundle.Txs {
+		var (
+			txHash = tx.Hash()
+			// update isRevertibleTx if transaction is found in list of reverting transaction hashes
+			isRevertibleTx = bundle.OriginalBundle.RevertingHash(txHash)
```

Comment:
This is actually expensive, and not needed most of the time - I wouldn't compute this unless necessary

Reply:
good catch, updated based on offline discussion